### PR TITLE
JAX-WS: Remove jaxws-2.3/jaxb-2.3 features and fat bucket repeats

### PR DESCRIPTION
**Pull request by neuwerk**
This pull request is the removal of the no-ship jaxws-2.3/jaxb-2.3 features and the relevant fat bucket repeats.

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by posmith** [comment]
L2 message review not required

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.jaxws.cdi_fat/fat/src/com/ibm/ws/jaxws/cdi/FATSuite.java [hunk 1]
```diff
@@ -27,8 +27,6 @@
 })
 public class FATSuite {
     @ClassRule
-    public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly())
-                    .andWith(FeatureReplacementAction.EE8_FEATURES().removeFeature("jaxws-2.2").addFeature("jaxws-2.3").withID("jaxws-2.3"))
-                    .andWith(FeatureReplacementAction.EE9_FEATURES().removeFeature("jaxws-2.3"));
+    public static RepeatTests r = RepeatTests.with(new EmptyAction()).andWith(FeatureReplacementAction.EE9_FEATURES());
```
Comment (jhanders34) [inline dev/com.ibm.ws.jaxws.cdi_fat/fat/src/com/ibm/ws/jaxws/cdi/FATSuite.java:30]: did you intend to remove the `.fullFATOnly()` from the EmptyAction?

**Review by jhanders34 — APPROVED** [review]
Approving with one question.

File: dev/com.ibm.ws.jaxws.cdi_fat/fat/src/com/ibm/ws/jaxws/cdi/FATSuite.java (hunk 1 above)
Comment (neuwerk) [reply dev/com.ibm.ws.jaxws.cdi_fat/fat/src/com/ibm/ws/jaxws/cdi/FATSuite.java:30]: Ah, yeah I do. I want both `jaxws-2.2` and `EE9` tests to run in lite mode for now.

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
Failure is unrelated to my changes.

**Comment by neuwerk** [comment]
#libby

**Pull request merged**